code/src/java/pcgen/gui2/coreview/CoreViewFrame.java

```diff
@@ -240,8 +238,8 @@ public void setPerspective(CorePerspective corePerspective)
 		@Override
 		public ListFacade<? extends TreeView<CoreViewNodeFacade>> getTreeViews()
 		{
-			DefaultListFacade<TreeView<CoreViewNodeFacade>> views =
-					new DefaultListFacade<TreeView<CoreViewNodeFacade>>();
+			DefaultListFacade<TreeView<CoreViewNodeFacade>> views
+					= new DefaultListFacade<TreeView<CoreViewNodeFacade>>();
```

Comment:
This doesn't look like our formatting guidelines.  Are you using the settings in standards?